content/docs/reference-glossary.md

```diff
@@ -41,11 +41,10 @@ React DOM uses camelCase property naming convention instead of HTML attribute na
 
 ```js
 const name = 'Clementine';
-ReactDOM.render(
+root.render(
```

Comment:
```js
const root = ReactDOM.createRoot(document.getElementById('root'));
```

Reply:
I'll just remove the render part, it's not relevant here, where the focus is on `class` vs `className`